hack/tools/tools.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2019 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import _ "github.com/corneliusweig/release-notes"
```

Comment:
this is not great. :( introduces a go package here.

let's just do something like this in the script, we already do it elsewhere:

```
temp="$(mkdir -p)"
trap -- 'rm -rf "${temp}"'
cd "$temp"
go mod init foo
go get github.com/corneliusweig/release-notes
# mv to '$(go env GOPATH)/bin' or something like that
```

Reply:
That's also fine with me.

Reply:
should be fixed now.